packages/popmotion-pose/src/dom/pose.ts

```diff
@@ -16,14 +17,23 @@ import {
 import { isPositional, convertPositionalUnits } from './unit-conversion';
 export { Action, Poser, ValueReaction, ColdSubscription };
 
+const getCurrent = (prop: string) => ({ elementStyler: Styler }) =>
```

Comment:
it's kinda a hack, currently dragEnd results in 2 property sets - one from `just` action and one from `applyAtEnd`, this part of the code tries to trick pose setter to handle it correctly by setting x/y on the poses, so it is what it is